proposals/URFC-TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+Title
+----
+* Author(s): [Author Name, Co-Author Name ...]
+* Approver: a11r
+* Status: {Draft, In Review, Final, Implemented}
+* Implemented in: <xDS/UDPA client, ...>
+* Last updated: [Date]
```

Comment:
Might be a good idea to specify this as YYYY-MM-DD, just to provide uniformity of date formats.